flixel/util/FlxCollision.hx

```diff
@@ -21,15 +24,17 @@ import flixel.tile.FlxTileblock;
 class FlxCollision 
 {
 	// Optimization: Local static vars to reduce allocations
-	private static var pointA:Point = new Point();
```

Comment:
I noticed a considerable improvement in PixelPerfectCollision demo (45fps vs 55fps, mostly gains from using faster Flx classes).

There is also some work to reduce unnecessary allocations in FlxMath functions.